commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v4_4_0/CommerceOrderAddressUpgradeProcess.java

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.internal.upgrade.v4_4_0;
+
+import com.liferay.commerce.model.CommerceOrder;
+import com.liferay.commerce.model.impl.CommerceOrderModelImpl;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.dao.jdbc.AutoBatchPreparedStatementUtil;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.service.ClassNameLocalService;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Alec Sloan
+ */
+public class CommerceOrderAddressUpgradeProcess extends UpgradeProcess {
+
+	public CommerceOrderAddressUpgradeProcess(
+		ClassNameLocalService classNameLocalService) {
+
+		_classNameLocalService = classNameLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		PreparedStatement ps = connection.prepareStatement(
+			"select * from CommerceAddress where classNameId = ?");
+
+		ps.setLong(
+			1, _classNameLocalService.getClassNameId(CommerceOrder.class));
+
+		ResultSet rs = ps.executeQuery();
```

Comment:
This doesn't need to be a `PreparedStatement` -- use a regular `Statement` instead. It only really makes sense to use a `ps` (IMO) if (1) we don't trust the input we're injecting into the SQL statement, so we should let the `ps` manage escaping it for us, or (2) we want to run the same statement multiple times with different values injected. Otherwise, in cases like this, just concatenate the statement with the `classNameId` and run it as a regular statement.

Reply:
Oh and also the statement and resultSet should be declared in a try-with-resources so it's properly closed.